### DISCUSSION TITLE
Form layout: alignement issue

### DISCUSSION THREAD

**Discussion by JulienDeBerlin** [quasarframework/quasar#9095] [General - Components / Directives / etc]
Hello, 

I'm trying to build a form and I have a small issue with the alignment of 2 elements in a raw, one of them being a q-toogle.
Here the [codepen link](https://codepen.io/JulienBD/pen/vYgwZwa) for demonstration
Can anyone give me a clue?

I also tried to align in a raw two form fields (one select and one input) and have exactly the same issue. 

Thanks for your help!
Julien

**Comment by ontwikkelfabriek** [comment]
here's a solution:
https://codepen.io/ontwikkelfabriek/pen/XWpweWP

I used this example from the docs:
https://quasar.dev/layout/grid/row#example--vertical-alignment

**Reply by JulienDeBerlin** [reply]
Thanks for your help!
